src/components/ProgressIndicator/_progress-indicator.scss

```diff
@@ -1,82 +1,306 @@
+    position: absolute;
+    left: 0;
+    top: $spacing-07;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: inherit;
+    height: 100%;
+    display: block;
+  }
+
+  .label {
+    position: relative;
+    font-family: $IBMPlexSans;
+    font-size: 14px;
+    color: #161616;
+    letter-spacing: 0.16px;
+    line-height: 18px;
+    cursor: default;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: max-content;
+
+    &-optional {
+      @extend .label;
+      font-size: $spacing-04;
+      color: #525252;
+      letter-spacing: 0.32px;
+      line-height: $spacing-05;
+      cursor: default !important;
+      margin-top: $spacing-01;
     }
 
-    .#{$prefix}--progress-step-button {
-      display: list-item;
-      padding: 0 $spacing-03;
-      text-align: left;
-      min-height: unset;
-      width: 100%;
+    transition: box-shadow $duration--fast-02 motion(standard, productive),
+      color $duration--fast-02 motion(standard, productive);
+  }
+
+  .line {
+    position: absolute;
+    flex: 1 1 auto;
+    width: calc(100% - 24px);
+    height: 2px;
+    background-color: #e0e0e0;
+    top: 11px;
+    left: 24px;
+
+    &-complete {
+      @extend .line;
+      background-color: #0f62fe;
     }
 
-    .#{$prefix}--progress-label {
-      max-width: calc(100% - 2.2rem);
-      position: relative;
+    &-sub {
+      @extend .line;
+      left: 16px;
+      width: calc(100% - 16px);
+
+      &-complete {
+        @extend .line-sub;
+        background-color: #0f62fe;
+      }
     }
+  }
 
-    .#{$prefix}--progress-optional {
-      margin-left: $spacing-08;
+  .step-button {
+    display: flex;
+    position: relative;
+    text-align: left;
+    width: 8rem;
+    height: 68px;
+    padding: 0;
+    margin: 0;
+
+    background-color: Transparent;
+    background-repeat: no-repeat;
+    cursor: default;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    outline: none;
+    border: none;
+
+    &.clickable {
+      .label {
+        cursor: pointer;
+      }
+    }
+
+    &:focus {
+      .label:not(.label-optional) {
+        box-shadow: 0 rem(3px) 0 0 $link-01;
+      }
+    }
+
+    &:hover {
+      .label:not(.label-optional) {
+        box-shadow: 0 rem(1px) $link-01;
+      }
+    }
+
+    &:active {
+      .label:not(.label-optional) {
+        box-shadow: 0 rem(3px) 0 0 $interactive-01;
+      }
     }
   }
 
-  .#{$prefix}--progress-optional {
-    margin-left: $spacing-07;
+  .hidden {
+    visibility: hidden;
   }
 
-  .#{$prefix}--progress-step svg {
-    margin-left: $spacing-03;
-    margin-right: $spacing-03;
+  .sub-step {
+    .icon {
+      margin-top: 4px;
+      margin-left: 0;
+    }
   }
 
-  .#{$prefix}--progress-line {
-    top: 0;
+  svg text {
+    font-size: 18px;
   }
-}
 
-.#{$prefix}--progress-line {
-  width: 100%;
-}
+  //-------States
+
+  .step-current {
+    &:not(.step-invalid) {
+      .main-step {
```

Comment:
Not sure if these are new classes but we should be using BEM syntax to create these 
```
.#{$prefix}--progress-line-step
.#{$prefix}--progress-line-step__current
.#{$prefix}--progress-line-step__invalid
etc. 
```